.rubocop_todo.yml

```diff
@@ -91,6 +91,7 @@ RSpec/VerifiedDoubles:
 RSpec/AnyInstance:
   Exclude:
     - 'spec/controllers/sufia/admin/admin_sets_controller_spec.rb'
+    - 'spec/controllers/sufia/trophies_controller_spec.rb'
```

Comment:
I'm not seeing why this is required.  I don't see any instances of this in the `trophies_controller_spec.rb`